tools/istio-clean-iptables/pkg/cmd/cleanup.go

```diff
@@ -81,7 +81,7 @@ func removeOldChains(cfg *config.Config, ext dep.Dependencies, iptV *dep.Iptable
 	}
 
 	// Must be last, the others refer to it
-	chains = []string{constants.ISTIOREDIRECT, constants.ISTIOINREDIRECT}
+	chains = []string{constants.ISTIOREDIRECT, constants.ISTIOINREDIRECT, constants.ISTIOOUTPUT}
```

Comment:
Yeah this is one of those cases where we shouldn't be duplicating cleanup stuff across separate binaries, really.

There is no real benefit to it and it's just another thing to keep in sync.

Reply:
Agreed. I saw that PR but seems like more discussions are occurring so this should resolve the issue for now and we can iterate a better procedure.

Reply:
fyi I have made #52918 (which relies the work done in #50328) to have VMs use the new istio-iptables --cleanup-option.